### PR TITLE
add .ci-operator.yaml file

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by coreydaley** [comment]
/assign @divyansh42 @mbharatk @apoorvajagtap
for lgtm

File: PR-Testing/Dockerfile-jenkins-test-new-plugin
Comment (apoorvajagtap) [inline PR-Testing/Dockerfile-jenkins-test-new-plugin:1]: Just to double-check, is the base image change (to `quay.io/coreydaley/jenkins:latest`) intentional?

Comment (coreydaley) [reply PR-Testing/Dockerfile-jenkins-test-new-plugin:1]: Nope, just pushed a fix, thanks for catching that

**Pull request merged**